.github/workflows/cmake.yml

```diff
@@ -198,8 +198,7 @@ jobs:
         -DUR_BUILD_ADAPTER_${{matrix.adapter.name}}=ON
         -DUR_DPCXX=${{github.workspace}}/dpcpp_compiler/bin/clang++
         -DUR_SYCL_LIBRARY_DIR=${{github.workspace}}/dpcpp_compiler/lib
-        -DUR_CONFORMANCE_TARGET_TRIPLES=${{matrix.adapter.triplet}}
-        ${{ matrix.adapter.name == 'HIP' && '-DAMD_ARCH=gfx1030' || '' }} 
+        ${{ matrix.adapter.name == 'HIP' && '-DAMD_ARCH=amdgcn' || '' }} 
```

Comment:
Ah, I see the HIP HW runner told you that too.

Reply:
I know I'm just messing around to see what we can get away with, sadly it doesn't look like we'll be able to automate the target ID stuff away

Reply:
Might possible by running `rocminfo` but that's out of scope for these changes.

Reply:
Oh I never turned the rocm stuff up while looking, even better `rocm_agent_enumerator` just gives you a list of architectures on the system with nothing else to clutter the output. I don't think it would be too onerous to see if we can detect that if `AMD_ARCH` isn't set, and exit with an error if we can't.